src/main/java/ca/on/oicr/gsi/dimsum/service/CaseService.java

```diff
@@ -307,6 +309,21 @@ public TableData<ProjectSummary> getProjects(int pageSize, int pageNumber,
     return data;
   }
 
+  public TableData<TestTableView> getTestTableViews(int pageSize, int pageNumber,
+      TestTableViewSort sort, boolean descending,
+      CaseFilter baseFilter, Collection<CaseFilter> filters) {
+    List<Case> cases = getCases(baseFilter);
+    TableData<TestTableView> data = new TableData<>();
+    data.setTotalCount(
+        cases.stream().flatMap(kase -> kase.getTests().stream()).distinct().count());
+    List<TestTableView> testTableViews = filterTestTableViews(cases, filters).distinct().toList();
```

Comment:
Is there a reason for distinct? I don't think there should ever be duplicates

Reply:
I'll remove them!